### ISSUE TITLE
Account page graph upgrade

### ISSUE THREAD

**Issue by monkeytypegeorge** [monkeytypegame/monkeytype#3786]

**Comment by ananyanair11** [comment]
@monkeytypegeorge @Miodec 
so does average of 10 and 100 imply average of the personal best 10 and 100 respectively?
Additionally, could you please clarify which line depicts what in the screenshot you have provided

**Comment by Miodec** [comment]
@ananyanair11 
No, average of 10 and 100 wold be just the average of results that are currently being filtered.
Not sure about your other question. What do you mean by which line.

@Ravina-Deogadkar were you interested in working on this? Do you have anything or are you waiting for approval?

**Comment by ananyanair11** [comment]
@Miodec 
By the lines I meant- in the screenshot provided above there's 3 blue lines of different shades. I wanted some clarity on what exactly those lines depict.

**Comment by ananyanair11** [comment]
Also, I don't quite understand on what basis an average of 10 or 100 could be taken unless it's the personal best 10 or 100.
Even in the image it says 'best avg 10' (I've marked them in red). 
It'd be great if I could get some info on that as well.
![chartmt](https://user-images.githubusercontent.com/85074812/206548444-a9490598-6483-424a-8cff-5c4fd5119cfa.png)

**Comment by Miodec** [comment]
Light blue is average of 10, medium blue is average of 100, dark blue is progression of best score.

**Comment by Miodec** [comment]
The average of 10 just means instead of drawing a point, draw the average of the last 10 points. That means the first 10 points will either be invisible (because there isn't enough points) or an average of 9,8,7 and so on.

**Comment by albertying** [comment]
Can I work on this?

**Comment by Miodec** [comment]
Sure, go ahead @albertying. Keep me updated often.
